File: 프로그래머스/0/181891. 순서 바꾸기/순서 바꾸기.cs
using System;
using System.Collections.Generic;
public class Solution {
    public int[] solution(int[] num_list, int n) {
        int[] answer = new int[num_list.Length];
        List<int> arr = new List<int>();
        for(int i = 0 ; i<num_list.Length;i++)
        {
            if(i<n)
                arr.Add(num_list[i]);
            else
            {
                arr.Insert(i-n,num_list[i]);
            }
        }
        answer = arr.ToArray();
        return answer;
    }
}

File: 프로그래머스/0/181891. 순서 바꾸기/순서 바꾸기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> num_list, int n) {
    vector<int> answer;
    for(int i = num_list.size() - 1; i >= 0; i--)
    {
        if(i > n - 1)
            answer.insert(answer.begin(), num_list[i]);
        else
            answer.insert(answer.begin() + num_list.size() - n, num_list[i]);
    }
    return answer;
}